schema/tables/login_password.yml

```diff
@@ -0,0 +1,12 @@
+name: login_password
+platforms:
+  - macOS
```

Comment:
Seems we use `darwin` on these yamls:
https://github.com/fleetdm/fleet/blob/39db759c8a073c4deb1347d3892de69b54c0219f/schema/tables/munki_info.yml#L1-L7

Reply:
@lucasmrod @sharon-fdm good catch.  Here's the implementation: (looks like it tolerates either):
https://github.com/fleetdm/fleet/blob/76417c3966cb7c45697b530cdfec4054834f7163/website/api/helpers/get-extended-osquery-schema.js#L145-L153


The code path that gets there, for reference: https://github.com/fleetdm/fleet/blob/76417c3966cb7c45697b530cdfec4054834f7163/website/scripts/build-static-content.js#L545-L547
(this is what merges on run)

fyi @zhumo and @eashaw are good people to ask if you have questions about this extended schema and aren't sure from the code.

Reply:
Thanks Mike.
I changed to darwin for the sake of uniformity.

Reply:
Yeah, darwin is macos because osquery's schema uses darwin, so it's for cross-compatibility.